src/components/pages/Login/LoginContainer.js

```diff
@@ -50,20 +49,23 @@ const LoginContainer = () => {
   }, []);
 
   return (
-    <StyledLogin>
+    <div className="root">
       <div
         className="background-image"
         aria-label="cosmetic background image"
       />
-      <div className="login-page">
-        <div id="sign-in-widget" aria-label="login form" />
-        <Button className="buttonStyles">
-          <Link to="/signup">
-            <p>Request to join</p>
+      <div className="form">
+        <div className="logo">
+          <img src={logo} alt="logo" width="350px" />
+        </div>
+        <div className="login-page">
+          <div id="sign-in-widget" aria-label="login form" />
+          <Link className="link-styles" to="/signup">
+            <p>Don't have an account? Register here</p>
```

Comment:
This message is such a good improvement from what we had before!

Reply:
Seconding this - and this code is much cleaner/more involved than what we had before!